src/engine.spec.ts

```diff
@@ -593,6 +593,58 @@ describe("Engine", () => {
       expect(engine.player(Player.Player1).data.advTechTiles).to.include(engine.advTechTiles[AdvTechTilePos.GaiaProject].tile);
       expect(engine.advTechTiles[AdvTechTilePos.GaiaProject].numTiles).to.equal(0);
     });
+
+    it("should allow Taklons to leech with +t freeIncome", () => {
```

Comment:
The test is in the "tech tile" portion of the test.

Seee https://travis-ci.org/donkeytech/gaia-engine/builds/402973639?utm_source=github_status&utm_medium=notification : 

```
    tech tiles
      ✓ should prevent picking the same tech tile twice (49ms)
      ✓ should allow to upgrade research area after building a RL, pick tech in nav
      ✓ should allow to get an advanced tech tiles when conditions are met (82ms)
      ✓ should allow Taklons to leech with +t freeIncome (42ms)
      ✓ should allow Taklons to leech with +t freeIncome (53ms)
```

Reply:
ok